screens/EventDetails/EventDetails.tsx

```diff
@@ -47,87 +48,89 @@ const EventDetails = ({ event }: EventDetailsProps) => {
 
   return (
     <View style={{ flex: 1 }}>
-      <ScrollView
-        style={[styles.container, styles.spacing]}
-        contentContainerStyle={{ flexGrow: 1 }}
-        nestedScrollEnabled={true}
-      >
-        <Title>{event.title}</Title>
-        <ProfileImageAndNameWithFriend
-          username={event.host.username}
-          userHandle={event.host.handle}
-          eventColor={event.color}
-          style={[styles.flexRow, { marginVertical: 24 }]}
-          imageStyle={styles.profileImage}
-          userFriendStatus="not-friends"
-        />
-        <View style={styles.iconSection}>
-          <TimeSection color={event.color} event={calenderEvent} />
-          <LocationSection
-            color={event.color}
-            coordinates={event.coordinates}
-            placemark={event.placemark}
+      <SafeAreaView>
```

Comment:
Use header instead to account for status bar, don't use the safeAreaView, unnecessary